api/src/main/java/org/commonjava/indy/conf/InternalFeatureConfig.java

```diff
@@ -0,0 +1,47 @@
+package org.commonjava.indy.conf;
+
+
+import org.commonjava.propulsor.config.annotation.ConfigName;
+import org.commonjava.propulsor.config.annotation.SectionName;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.io.File;
+import java.io.InputStream;
+
+@ApplicationScoped
+@SectionName( InternalFeatureConfig.SECTION_NAME )
+public class InternalFeatureConfig implements IndyConfigInfo {
+
+    public final static String SECTION_NAME = "ssl_validation";
```

Comment:
Let's change the section name here to something like `[internal]` since this isn't SSL-specific.